ros_buildfarm/templates/status/blocked_releases_page.html.em

```diff
@@ -0,0 +1,111 @@
+        <script language="JavaScript">encoded_query_a_tag("label=\"UNBLOCKED\"", "Repositories which can be released", "releasable")</script>,
+        <script language="JavaScript">encoded_query_a_tag("label=\"UNBLOCKED_BLOCKING\"", "Repositories which can be released and are preventing others from being released", "releasable and blocking")</script>,
+        <script language="JavaScript">encoded_query_a_tag("label=\"UNBLOCKED_UNBLOCKING\"", "Repositories which can be released and are not preventing others from being released", "releasable and not blocking")</script>,
+        <script language="JavaScript">encoded_query_a_tag("id=\"metapackages\"", "Repositories which are dependencies of the desktop-full metapackage", "desktop-full")</script>
+      </p>
+      <p id="search-count"></p>
+    </form>
+  </div>
+
+  <div class="table-div">
+    <table>
+      <thead>
+        <tr>
+          <!-- warning: if titles run onto two lines, first row of table data may be affected -->
+          <th class="sortable"><div>Repository</div></th>
+          <th class="sortable"><div>Version</div></th>
+          <th class="sortable">
+            <div title="Number of repositories which are directly blocking this one by not being released">
+              # blocking release
+            </div>
+          </th>
+          <th class="sortable">
+            <div title="Repositories which are directly blocking this one by not being released">
+              Blocking repos
+            </div>
+          </th>
+          <th class="sortable">
+            <div title="Maintainers of the repositories which are directly blocking this one by not being released">
+              Maintainers of blocks
+            </div>
+          </th>
+          <th class="sortable">
+            <div title="Number of repositories which are being directly or indirectly blocked by this one not being released">
+              # recursively blocked
+            </div>
+          </th>
+          <th class="sortable">
+            <div title="Number of repositories which are being directly blocked by this one not being released">
+              # directly blocked
+            </div>
+          </th>
+          <th class="sortable">
+            <div title="Repositories which are being directly blocked by this one not being released">
+              Directly blocked repos
+            </div>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <script type="text/javascript">window.tbody_ready();</script>
+    @[for row in repos_data]@
+      <tr>
+    @[for col in [
+      'name',
+      'version',
+      'num_repos_blocked_by',
+      'repos_blocked_by',
+      'maintainers_of_repos_blocked_by',
+      'num_repos_recursively_blocked',
+      'num_repos_blocked',
+      'repos_blocked',
+      ]]@
+      <td @(' class="thincol"' if col in ['version',] or col.startswith('num') else '')>@(row[col])</td>
+    @[end for]@
+    </tr>
+  @[end for]@
+    </tbody>
```

Comment:
perfect, thanks for that. I was only moving the `@` to the beginning of the line so was getting errors. have requested membership in that team so I get pinged in the future